sass/components/_day-picker.scss

```diff
@@ -19,7 +19,16 @@
         }
 
         .DayPicker-Day {
-            border-radius: $border-rad * 2;
+            min-width: 32px;
```

Comment:
same here ... IMO changin the base component is pretty unsafe, sice it might have a big impact on other components

Reply:
@michelengelen The base component for the day picker has changed, so we want this change reflected elsewhere too, which is why we have changes directly to the main component.
You can see the component here:
https://www.figma.com/file/0XY1PaiNrMXP2Z2E4gcsz7/Components---Date-Picker?node-id=86%3A689